instrumentation/logback/logback-appender-1.0/javaagent/src/test/groovy/LogbackTest.groovy

```diff
@@ -56,12 +56,12 @@ class LogbackTest extends AgentInstrumentationSpecification {
       assertThat(log.getSeverity()).isEqualTo(severity)
       assertThat(log.getSeverityText()).isEqualTo(severityText)
       if (exception) {
-        assertThat(log.getAttributes().size()).isEqualTo(5)
+        assertThat(log.getAttributes().size()).isEqualTo(5 + 4) // 4 code attributes
         assertThat(log.getAttributes().get(SemanticAttributes.EXCEPTION_TYPE)).isEqualTo(IllegalStateException.getName())
         assertThat(log.getAttributes().get(SemanticAttributes.EXCEPTION_MESSAGE)).isEqualTo("hello")
         assertThat(log.getAttributes().get(SemanticAttributes.EXCEPTION_STACKTRACE)).contains(LogbackTest.name)
       } else {
-        assertThat(log.getAttributes().size()).isEqualTo(2)
+        assertThat(log.getAttributes().size()).isEqualTo(2 + 4)  // 4 code attributes
```

Comment:
Can you add code attributes assertions to the javaagent test too?

Reply:
Because of the nature of the Groovy test, the attribute values are not very readable:
```
file = NativeMethodAccessorImpl.java
codeClass = jdk.internal.reflect.NativeMethodAccessorImpl
method = invoke0
```

[The attribute values are checked in a library test.](https://github.com/jeanbisutti/opentelemetry-java-instrumentation/blob/5db9dedae25f99e7074774214d64e1d6aab5717b/instrumentation/logback/logback-appender-1.0/library/src/test/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppenderConfigTest.java#L125)

Reply:
Eh, forgot that was groovy. Nevermind then 👍

Reply:
ya, it would be nice to have validation that the attributes are working inside of the Java agent as well, but can get to that later, I've opened #6604 to track